src/main/java/Event.java

```diff
@@ -11,8 +11,8 @@ public Event(String inLine){
         }
```

Comment:
In line 11, regarding the coding standard, the { should be put in the same line of "public".